### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/AutoConfigurationImportSelector.java [hunk 1]
```diff
@@ -412,8 +412,8 @@ public void process(AnnotationMetadata annotationMetadata,
 				DeferredImportSelector deferredImportSelector) {
 			Assert.state(
 					deferredImportSelector instanceof AutoConfigurationImportSelector,
-					String.format(
-							"AutoConfigurationImportSelector only supports %s implementations, got %s",
```
Comment (snicoll) [inline spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/AutoConfigurationImportSelector.java]: This isn't a typo but a conscious decision. `AutoConfigurationGroup` is an internal concept while the one in the exception is the public concept. 

What do you think?

Comment (izeye) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/AutoConfigurationImportSelector.java]: @snicoll Thanks for the clarification! When I saw it, "AutoConfigurationImportSelector only supports AutoConfigurationImportSelector implementations" sounds a bit weird and it's in `AutoConfigurationGroup.process()`, so I was trying to change it to `AutoConfigurationGroup` which sounds more natural but noticed that the class is `private` scope as you mentioned. I wasn't sure with this change at that time but with your comment, it'd be better to be reverted for now. I'll revert it soon.

Comment (izeye) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/AutoConfigurationImportSelector.java]: @snicoll I reverted it and force-pushed.

Comment (snicoll) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/AutoConfigurationImportSelector.java]: You got a valid point in any case. I'll reword that exception message

**Comment by snicoll** [comment]
Thanks again @izeye - I've hopefully improved the message in f7032bd

**Pull request closed without merging**